### PR TITLE
docs: update getting started

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

This pull request updates getting started and breaks it up into multiple sections including the following:

- quick start (running a single node network for testing with delegating example)
- live networks (list live networks and using the CLI to interact with live networks)
- running a validator node (using cosmovisor and validator specifc instructions)
- running a full node (using cosmovisor and full node specific instructions)
- prerequisites (information about software and hardware prerequisites)

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [x] followed the [documentation writing guidelines](https://github.com/cosmos/cosmos-sdk/blob/master/docs/DOC_WRITING_GUIDELINES.md)
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct `docs:` prefix in the PR title
- [ ] confirmed all author checklist items have been addressed 
- [ ] confirmed that this PR only changes documentation
- [ ] reviewed content for consistency
- [ ] reviewed content for thoroughness
- [ ] reviewed content for spelling and grammar
- [ ] tested instructions (if applicable)

### CODE REVIEW

File: docs/getting-started/running-a-full-node.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Running a Full Node
+
+This document provides instructions for running a full node for a live network.
```
Comment (clevinson) [inline docs/getting-started/running-a-full-node.md:3]: Maybe good to specify here "live network for our mainnet or devnet"

File: docs/getting-started/running-a-full-node.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+Check out `v1.0.0`:
+```
+git checkout v1.0.0
+```
+
+Install `regen` binary:
+```
+make install
```
Comment (clevinson) [inline docs/getting-started/running-a-full-node.md:34]: We should split out devnet to have `EXPERIMENTAL=true`

Comment (clevinson) [reply docs/getting-started/running-a-full-node.md:34]: Actually we should also specify the tag....

**Review by clevinson — APPROVED** [review]
Nice see some love on Regen Ledger docs! thx for the updates.  Few small nits. Otherwise lgtm.

File: docs/getting-started/running-a-full-node.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+```
+PERSISTENT_PEERS="b2679a74d6bd9f89a3c294c447d6930293255e6b@18.220.101.192:26656"
+sed -i '/persistent_peers =/c\persistent_peers = "'"$PERSISTENT_PEERS"'"' ~/.regen/config/config.toml
+```
+
+## Start Node
+
+Start node:
+```
+regen start
```
Comment (anilCSE) [inline docs/getting-started/running-a-full-node.md:110]: Let's add another option to use system service.

File: docs/getting-started/README.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+In this case, `node` is the name (or "moniker") of the node and `test` is the chain ID. Feel free to change these values but make sure to use the same value for `chain-id` in the following steps.
+
+## Update Genesis
+
+When the node was initialized, a `genesis.json` file was created within the `config` directory. In this section, you will be adding two genesis accounts (accounts with an initial token balance) and a genesis transaction (a transaction that registers the validator account in the validator set).
+
+Add `validator` account to `genesis.json`:
+```
+regen add-genesis-account validator 5000000000stake --keyring-backend test
```
Comment (anilCSE) [inline docs/getting-started/README.md:81]: can we use different denom than `stake`? may be `uregen`? We can update genesis to use custom denom after `init` using the following command:
```sh
sed -i "s/stake/$DENOM/g" $DAEMON_HOME/config/genesis.json
```

**Review by anilCSE — APPROVED** [review]
lgtm, some nits

**Pull request merged**